### PR TITLE
Change the JSON renderer to indent conditionally

### LINKED ISSUE #14: Have the JSON renderer pretty print when the c2c_pretty parameter is set

### PR DESCRIPTION
**Pull request by pvalsecc**
Closes #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -288,6 +288,12 @@ c2c.disable_exception_handling = 1
 ```
 
 
+# JSON pretty print
+
+The standard JSON renderer is replaced by a version that indents the rendered JSON if the `?c2c_pretty`
```
Comment (sbrunner) [inline README.md]: Why not just `pretty`?

Comment (pvalsecc) [reply README.md]: To avoid potential clashes with the route's parameters. But if you think it's not needed, I'll change it to just pretty. Just tell me.

**Comment by romanzoller** [comment]
[Some people](http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#pretty-print-gzip) recommend to always pretty print, and claim that e.g. GitHub does this, the idea being that whitespace overhead is negligible if the response is gzipped. Thoughts?

This change that added some lines of `?c2c_pretty` brought me here: https://github.com/camptocamp/saccas_suissealpine/pull/862

**Comment by gberaudo** [comment]
Yes, would be interesting to have figures.

**Comment by pvalsecc** [comment]
Body size, gzip encoded:

* https://dev.suissealpine.sac-cas.ch/api/c2c/versions.json: 1012 bytes
* https://dev.suissealpine.sac-cas.ch/api/c2c/versions.json?c2c_pretty: 960 bytes

Now, regarding the rendering perfs (average over 10 measures from the c2c network):

* https://dev.suissealpine.sac-cas.ch/api/c2c/versions.json: 0.3374s
* https://dev.suissealpine.sac-cas.ch/api/c2c/versions.json?c2c_pretty: 0.3424s

**Pull request merged**